index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+<meta charset=“UTF-8”>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Oswald:wght@300&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="./styles/reset.css">
+<link rel="stylesheet" href="./styles/style.css">
+<title>My osom site</title>
+</head>	
+
+<body>
+<div class="background_box">
+		<div class="main_box">
+
+		<div class="top_box">
+			<div class="logo">
+				<p>Logo</p>
+			</div>
+			<div class="button">
+				<p>Home</p>
+			</div>
+			<div class="button">
+				<p>About</p>
+			</div>
+			<div class="button">
+				<p>Contact</p>
+			</div>
+		</div>
+
+		<div class="image_box">
+			<div class="main_text">
+				<p>My osom site</p>
+			</div>
+		</div>
+
+		<div class="article_box">
+			<h1>First article</h1>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum </p><br>
+				<p>02.04.2022</p>
+		</div>
+		<div class="collapsing_box">
+		</div>
```

Comment:
jest to ok, ale presonalnie produkcyjnie chyba bym nie uzywała takich pustych diviów do stylowania, raczej bym owrapowała te boxy jeszcze w jakieś kontenery :) na potrzebe zadania ok